spec/index.bs

```diff
@@ -6693,6 +6693,7 @@ following members:
 
                                                 - [=Assert=] |this|.{{GPUObjectBase/[[device]]}}.{{device/[[features]]}} does not [=list/contain=] {{GPUFeatureName/"core-features-and-limits"}}.
                                                 - |texture|.{{GPUTexture/textureBindingViewDimension}} must be equal to |textureView|.{{GPUTextureViewDescriptor/dimension}}.
+                                            Note: this validation only applies to a user-specified textureBindingViewDimension. If no value is provided, the texture's textureBindingViewDimension is set as described in {{GPUDevice/createTexture()}}. That algorithm cannot produce invalid values, so the above validation is not required.
```

Comment:
I was thinking of a different spot for the note - I hadn't notice this one, but this is supposed to be pointing to the GPUTexture attribute, not the GPUTextureDescriptor member, so I think it's fine without a note.